src/ip/QosConfig.cc

```diff
@@ -533,7 +533,11 @@ Ip::Qos::setSockTos(const int fd, tos_t tos, int type)
     // unless we pass it a 32-bit variable storing 8-bits of data.
```

Comment:
Please move this comment inside the `#else` statement were FreeBSD code will now live.

Changing our `int` to `int32_t` to match the claim in the comment is probably out of this PR scope.